.github/workflows/e2e-test-weekly.yml

```
@@ -343,3 +343,18 @@ jobs:
     uses: ./.github/workflows/e2e-windows.yml
     with:
       scheduled: ${{ github.event_name == 'schedule' }}
+
+  e2e-tf-module:
+    name: Apply Constellation Terraform module
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+    secrets: inherit
+    uses: ./.github/workflows/e2e-test-tf-module.yml
+    with:
+      cloudProvider: "aws"
+      region: "eu-west-1"
+      zone: "eu-west-1b"
+      name: "e2eTfAws"
```

```suggestion
      name: "e2eTfAws"
```
Can we parametrize this to a workflow run, e.g. by incorporating the run ID in the naming, similar to what we do with Azure resource groups? Otherwise, runs failing to terminate the resource may cause an avalanche of other failing runs due to naming collisions.

---

done, i had to shorten the logic though due to the 10 char limit on AWS.